common/persistence/cassandra/cassandraPersistenceUtil.go

```diff
@@ -1817,9 +1825,40 @@ func createShardInfo(
 		info.ClusterReplicationLevel = make(map[string]int64)
 	}
 
+	if info.ClusterTransferProcessingQueueState == nil {
+		info.ClusterTransferProcessingQueueState = map[string]p.ProcessingQueueState{}
+		for cluster, ackLevel := range info.ClusterTransferAckLevel {
+			processingQueueState := createProcessingQueueState(ackLevel)
+			info.ClusterTransferProcessingQueueState[cluster] = processingQueueState
+		}
```

Comment:
warning: cyclomatic complexity 61 of function createWorkflowExecutionInfo() is high (> 15) (gocyclo)